src/utils/time-format.js

```diff
@@ -33,15 +33,86 @@ function toHHMMSS(input: number): string {
  */
 function toSecondsFromHHMMSS(input: string): number {
```

Comment:
if you have separate cases of HHMM and HHMMSS, i think it's better to split into 2 separate functions, 
like toSecondsFromHHMM and toSecondsFromHHMMSS